Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN pip install --upgrade pip virtualenv
 # Install the Google-built interpreters
 ADD interpreters.tar.gz /
 
+# Add Google-built interpreters to the path
+ENV PATH $PATH:/opt/python3.5/bin
```

Comment:
Paranoia from bitter experience.  E.g. somehow a file named "sh" or "true" ends up in /opt/python3.5/bin and everything goes crazy.  I always add things to the end of $PATH.

Tangential rant, this is why Python's handling of $PYTHONPATH is so nutty.  What, you have a directory named "os" in the current working directory?  Let me helpfully break everything for you.